.travis.yml

```diff
@@ -14,14 +14,18 @@ env:
     # PHP 7.2
     - PHP_VERSION=7.2 PSR_HTTP_PROVIDER=nyholm SYMFONY_VERSION=3.4.0
     - PHP_VERSION=7.2 PSR_HTTP_PROVIDER=nyholm SYMFONY_VERSION=4.2.0
+    - PHP_VERSION=7.2 PSR_HTTP_PROVIDER=nyholm SYMFONY_VERSION=4.3.0
     - PHP_VERSION=7.2 PSR_HTTP_PROVIDER=zendframework SYMFONY_VERSION=3.4.0
     - PHP_VERSION=7.2 PSR_HTTP_PROVIDER=zendframework SYMFONY_VERSION=4.2.0
+    - PHP_VERSION=7.2 PSR_HTTP_PROVIDER=zendframework SYMFONY_VERSION=4.3.0
```

Comment:
@HypeMC I remember running the test suite locally with SF 4.3 and it failed. The reason of the failure was either https://github.com/symfony/symfony/pull/31881 and/or https://github.com/symfony/symfony/pull/31880 . Those were breaking changes that were released with Symfony 4.3.0 and which were fixed in 4.3.1 in those PRs so we should use 4.3.1 for our CI instead of 4.3.0.

Reply:
@X-Coder264 No need since te command is `dev/bin/php composer require symfony/symfony:~${SYMFONY_VERSION} --ansi --no-update` so you get `~4.3.0` which will install the latest patch version.

Reply:
Oh sorry, I didn't see the tilde character there 😄